app/models/portfolio.rb

```diff
@@ -22,7 +22,7 @@ def add_portfolio_item(portfolio_item)
   attribute :metadata, ActiveRecord::Type::Json.new
 
   def metadata
-    user = UserContext.new(Insights::API::Common::Request.current!, nil)
+    user = Thread.current[:user]
```

Comment:
@eclarizio Should we raise an exception if we can't find the user or skip the user capabilities call.
Since someone could just start a rails console and try to get a Portfolio object and it's going to fail.
So either

(1) If the user is missing skip the metadata and return {}
(2) We can raise an exception if the user is missing